nicsdru_unity_theme.theme

```diff
@@ -136,3 +136,12 @@ function nicsdru_unity_theme_preprocess_views_view(&$variables) {
     $variables['exposed']['query']['#title_display'] = 'invisible';
   }
 }
+
+/**
+ * Implements hook_theme_preprocess_image().
+ */
+function nicsdru_unity_theme_preprocess_image(&$variables) {
+  if (!isset($variables['attributes']['alt'])) {
+    $variables['attributes']['alt'] = '';
```

Comment:
Would it better to check/handle the empty value in a template (if it exists) rather than pass in an empty value in the preprocess hook?